.github/workflows/release-the-blueprint.yaml

```diff
@@ -0,0 +1,42 @@
+name: Release the blueprint
+
+on:
+  push:
+    tags:
+      - "*"
+
+permissions:
+  contents: write
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v4
+      - name: Determine variables
+        id: determine-variables
+        run: |
+          folder=${GITHUB_REF#*/tags/}
+          folder=${folder%-*}
+          version=${GITHUB_REF##*/}
+          version=${version//[!0-9.]/}
+
+          release_name=$(echo "${folder//-/ }" | sed -e 's/\b\(.\)/\u\1/g')
```

Comment:
`sed -e "s/\b\(.\)/\u\1/g;s/Brians/Brian's/g"`

or even

`echo "$folder" | sed -e "s/-/ /g;s/\b\(.\)/\u\1/g;s/Brians/Brian's/g"`

Reply:
Resolved in https://github.com/bcwhite-code/brians-blueprints/pull/27/commits/42a4b177579ec1110a3dc9d9f57795408c057dc4